src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java

```diff
@@ -305,7 +305,7 @@ private void addTag(String text, int line) {
          * @param filter the {@code SuppressWithNearbyCommentFilter} with the context
          * @throws IllegalArgumentException if unable to parse expanded text.
          */
-        public Tag(String text, int line, SuppressWithNearbyCommentFilter filter) {
+        /* package */ Tag(String text, int line, SuppressWithNearbyCommentFilter filter) {
```

Comment:
what is a story to make it `package` ? why not public or private ?

Reply:
`public` triggers `redundant modifier`
`private` forces the compiler to generate package private accessors.

We discussed this recently in #6813